src/lib/holds.js

```diff
@@ -3,7 +3,7 @@
 const InsufficientFundsError = require('../errors/insufficient-funds-error')
 
 function adjustBalance (account, amount, transaction) {
-  const updateSQL = 'UPDATE "accounts" SET "balance" = "balance" + ? WHERE "name" = ?'
+  const updateSQL = 'UPDATE "L_ACCOUNTS" SET "BALANCE" = "BALANCE" + ? WHERE "NAME" = ?'
```

Comment:
Can you remind me why this was raw SQL? Now that we have the persistence layer in db/accounts, it seems like we could add a method to that file that uses knex.